homeassistant/components/dlink/sensor.py

```diff
@@ -0,0 +1,65 @@
+"""Support for D-Link Power Plug Sensors."""
```

Comment:
Please keep this file out for the sensor PR 🙏🏻 

This will keep this PR nice and scoped towards adding the coordinator, and then the followup PR will be nicely scoped towards adding the sensors!